src/txn/txn_rollback_to_stable.c

```diff
@@ -277,23 +319,32 @@ __rollback_row_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW
          * update chain. Also it confirms that history store doesn't contains any newer version than
```

Comment:
`current version stop timestamp`  to `current version stop timestamp and transaction id`.